.markdownlint.yaml

```diff
@@ -0,0 +1,2 @@
+MD013:
+  line_length: 1000
```

Comment:
I don't think we need both yaml and json, it should be one or the other.

Reply:
TIL about the existing json file. I've removed the YAML one.